doc/src/compute_pressure_cylinder.txt

```diff
@@ -0,0 +1,64 @@
+"LAMMPS WWW Site"_lws - "LAMMPS Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Section_commands.html#comm)
+
+:line
+
+compute pressure/cylinder command :h3
+
+[Syntax:]
+
+compute ID group-ID pressure/cylinder zlo zhi Rmax bin_width :pre
+
+ID, group-ID are documented in "compute"_compute.html command
+pressure/cylinder = style name of this compute command
+zlo = minimum z-boundary for cylinder
+zhi = maximum z-boundary for cylinder
+Rmax = maximum radius to perform calculation to
+bin_width = width of radial bins to use for calculation :ul
+
+[Examples:]
+
+compute 1 all pressure/cylinder -10.0 10.0 15.0 0.25 :pre
+
+[Description:]
+
+Define a computation that calculates the pressure tensor of a
+system in cylindrical coordinates, as discussed in "(Addington)"_#Addington1.
+This is useful for systems with a single axis of rotational symmetry,
+such as cylindrical micelles or carbon nanotubes. The compute splits 
+the system into radial, cylindrical-shell-type bins of width bin_width, centered at x=0,y=0,
+and calculates the radial (P_rhorho), azimuthal (P_phiphi), and axial (P_zz) components of the configurational pressure
+tensor. The local density is also calculated for each bin, so that
+the true pressure can be recovered as P_kin+P_conf=density*k*T+P_conf.
+The output is a global array with 5 columns; one each for bin radius,
+local number density, P_rhorho, P_phiphi, and P_zz. The number of 
+rows is governed by the values of Rmax and bin_width. Pressure
+tensor values are output in pressure units.
+
+[Output info:]
+
+This compute calculates a global array with 5 columns and Rmax/bin_width
+rows.
+
+The values calculated by this compute are
+"intensive".  The pressure values will be in pressure
+"units"_units.html. The number density values will be in 
+inverse volume "units"_units.html.
+
+[Restrictions:] none
```

Comment:
Please add under restrictions, that this compute computes the contributions from pair styles (i.e. non-bonded interactions) only and that it requires functionality not available by most manybody pair styles. see `compute group/group`. There also is the issue of how pair styles for use with a kspace style are handled. it looks to me, that those are not compatible to this as well. Finally, it seems that this compute, that this does not consider kinetic contribution (by default) unlike other pressure compute styles in LAMMPS. so it is probably advisable to document this as well.

Reply:
I've clarified the restrictions and output in the documentation now, so hopefully everything is explicit enough now. Thanks!